modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -28,11 +28,11 @@ const SidebarBody = ({children, className}) => {
 	);
 };
 
-const SidebarHeader = ({children, title}) => {
+const SidebarHeader = ({actionsSlot, children, className, title}) => {
 	const {onClose} = useContext(SidebarContext);
 
 	return (
-		<section className="sidebar-header">
+		<section className={classNames('is-sticky sidebar-header', className)}>
```

Comment:
Hi @beltranrengifo , what do you think about applying the is-sticky class only if we have the FF enabled?